content/messaging/agent-chat-api/index.mdx

```diff
@@ -351,6 +351,12 @@ The **Message** event contains text message to other chat users.
 | `type`                | yes     | `string`  | `message`                                                           |
 | `recipients`          | yes     | `string`  | Possible values: `all` (default), `agents`                          |
 | `properties`          | no      | `object`  | [Properties](#properties)                                           |
+| `postback`            | no      | `object`  | Indicates that the message event was generated from a rich message event. |
```

Comment:
how about
`Indicates that the message event was generated as the response to a rich message event.`
?

Reply:
why as the response? what do you mean?

Reply:
If you ask me a closed question in the form of a rich message with some buttons and I click one of them then the postback will tell you which one I clicked (what my response is).

Reply:
I see. What about: 
Indicates that the message event was generated in response to a rich message event.

I believe the meaning is the same but sounds a bit more natural.

Reply:
ok